content/authoring/tutorials/create-first-kata.mdx

```diff
@@ -32,175 +32,192 @@ To create a new kata, you need to earn the ["Create Kata" privilege](/gamificati
   }}
 />
 
-## How to make a good kata?
+## How to create a good kata
 
-Everyone enjoys good, wellcrafted kata, and no one likes to struggle with poorly authored challenges. However, especially for unexperienced authors, it's often difficult to tell what is good, what is bad, or why. There's a few aspects which can drastically influence experience of other users with your kata and, as a result, feedback it will receive.
+While everyone enjoys an interesting and well-crafted kata, no one likes to struggle with poorly authored challenges. That said, it is often difficult to identify good and bad kata qualities and determine what sets them apart, especially for unexperienced authors. There are a few factors that have a drastic influence on the enjoyment other users will experience solving your kata and, as a result, the feedback it will receive.
 
-### Kata Idea and Duplicates
+### Kata ideas and duplicates
 
-At the time of writing, there are already **over 6500** approved kata, and almost 2500 beta kata. It's a vast red ocean out there - if you're still sticking to ideas like Fizzbuzz, Fibonacci numbers, or Caesar/ROT cipher, it's pretty much 99.999999...% chance that someone has done this before you.
+At the time of writing, there are already **~7000** approved kata and over 2000 kata in beta status. With such a vast selection of kata, chances are high that your idea may not be unique - popular topics like FizzBuzz, Fibonacci numbers and Rotation Ciphers are virtually guaranteed to have been covered already.
 
-This is bad because creating a kata about them again constitutes a **duplicate** kata, and we certainly don't want 100 Fizzbuzz kata out in the wild. When a duplicate is found it will be retired, which basically means it gets taken out of the kata catalog.
+The reason we mention this is because we don't need dozens of **duplicate** FizzBuzz kata. Whenever a [kata is identified as a duplicate](https://github.com/Codewars/content-issues/wiki/List-of-Possible-Duplicate-Kata) (and it usually will), it will be retired and thus effectively removed the kata catalog.
```

Comment:
```suggestion
The reason we mention this is that we don't need dozens of **duplicate** FizzBuzz kata. Whenever a [kata is identified as a duplicate](https://github.com/Codewars/content-issues/wiki/List-of-Possible-Duplicate-Kata) (and it usually will), it will be retired and thus effectively removed the kata catalog.
```